libtft.py

```diff
@@ -380,3 +396,22 @@ def cmd_hash_object(args):
     with open(args.path, "rb") as fd:
         sha = object_hash(fd, args.type.encode(), repo)
         print(sha)
+
+def cmd_checkout(args):
+    """Bridge function to checkout an existing repository."""
+    repo = repo_find(args.path)
```

Comment:
no need to pass an argument here. args.path, also, represents where we want to checkout out old commit worktree from what i understand. Please remove arg